templates/includes/pagination.html

```diff
@@ -0,0 +1,17 @@
+{% if is_paginated %}
+  <ul class="pagination">
+    {% if page_obj.has_previous %}
+      <li class="page-item">
+        <a href="?page= {{ page_obj.previous_page_number }}" class="page-link">prev</a>
```

Comment:
There's an extra space after `page=` in the `href` attribute. Consider removing it to ensure the URL is correctly formatted.